mmv1/templates/terraform/examples/dataplex_entry_basic.tf.tmpl

```diff
@@ -0,0 +1,20 @@
+resource "google_dataplex_entry_group" "{{index $.Vars "entry_group_name"}}" {
```

Comment:
Terraform resource names only need to be unique within a single config file, and for examples usually only the primary resource ID needs to be specified by a variable, so this could be:

```
resource "google_dataplex_entry_group" "basic-entry-group" {
```

(the actual API resource name needs to be unique though, so `entry_group_id` needs to continue to reference the variable)